cms/standard_pages/models.py

```diff
@@ -38,3 +47,76 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class MethodologyPage(BasePage):  # type: ignore[django-manager-missing]
```

Comment:
nitpick: my preference would be to have this in a separate app.

Reply:
Done in: [bc5136f](https://github.com/ONSdigital/dis-wagtail/pull/57/commits/bc5136f9ab2afd0e63446f3064deefe7b9a6acdd)